recipe/bld.bat

```diff
@@ -24,11 +24,9 @@ if errorlevel 1 exit /b 1
 
 
 :: Perform tests.
-::  echo "Testing..."
-::  ninja test
-::  path_to\test
-::  ctest -VV --output-on-failure
```

Comment:
Since I cannot commit something proper at this time, this gave the opportunity to switch to this line.
I can see that some tests somehow rely on a python package that is missing.